skyve-war/src/main/java/modules/admin/Subscription/SubscriptionService.java

```diff
@@ -175,7 +182,9 @@ public static boolean anonymouslyCheckSubscriptionPublicUser(Persistence persist
 	 * @param receiverIdentifier
 	 * @return
```

Comment:
Clean up Javadoc; these parameters and return types should have proper descriptions. Review the method description, which doesn't provide enough information to understand what this method actually does without reading the code. E.g., what does anonymously check mean?